templates/mixins/attachments.jade

```diff
@@ -13,7 +13,7 @@ mixin attachments(url, attachments, canEdit)
             tbody
                 for attachment in attachments
                     tr
-                        td #{attachment.type}
+                        td #{attachment.image?'Image':attachment.type}
```

Comment:
This could be done in JS and doing so would allow us to create other broad categories.

Reply:
Agreed.  It is a bit of clugge.  The format types for attachments need a bit of work, especially when it comes to EDAM types.